posthog/api/capture.py

```diff
@@ -677,6 +680,23 @@ def replace_with_warning(event: dict[str, Any]) -> dict[str, Any] | None:
         return None
 
 
+def sample_replay_data_to_object_storage(event: dict[str, Any], random_number: float) -> None:
+    """
+    the random number is passed in to make testing easier
+    both the random number and the sample rate must be between 0 and 0.01
```

Comment:
Usually, sample rate is more human-readable when it's between 0...1, so 0.1 is 10% and 1 is 100%, It's not a blocker, just the standard I've seen across different tools.

Reply:
Ah, yep.. so this is still between 0 and 1 but clamped to 0.01
I don't want to upload 100s of messages a minute if someone misconfigures this value